src/locale/zh-CN.js

```diff
@@ -34,6 +34,7 @@ const map = {
     api: 'API',
     examples: '示例',
     blog: '博客',
+    storybook: 'Storybook (WIP)',
```

Comment:
It might be nice to run Work In Progress through google translate and copy paste the chinese output here into the brackets.

Reply:
Agreed sorry, this was laziness on my part